components/toast/toast.html

```diff
@@ -0,0 +1,160 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="../../assets/Wings.svg" />
+    <link rel="stylesheet" type="text/css" href="../../utilities/doc.css" />
+    <title>Wings UI | Toast</title>
+  </head>
+  <body>
+    <header id="home-header" class="flex-row flex-space-btw">
+      <h1 class="home-header-title"><a href="/">WINGS UI</a></h1>
+      <nav class="home-nav flex-row">
+        <ul class="home-nav-list ls-none">
+          <li class="home-nav-item">
+            <a class="home-nav-link" href="/">Home</a>
+          </li>
+          <li class="home-nav-item">
+            <a class="home-nav-link nav-link-active" href="../alert/alert.html"
+              >Docs</a
+            >
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main id="doc-main" class="flex-row">
+      <aside class="doc-sidebar">
+        <ul class="sidebar-list ls-none mp-m">
+          <strong class="sidebar-header font-m">Components</strong>
+          <li class="sidebar-item mp-m">
+            <a href="../alert/alert.html" class="sidebar-link">Alert</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../avatar/avatar.html" class="sidebar-link">Avatar</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../badge/badge.html" class="sidebar-link">Badge</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../button/button.html" class="sidebar-link">Button</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../card/card.html" class="sidebar-link">Card</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../grid/grid.html" class="sidebar-link">Grid</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../image/image.html" class="sidebar-link">Image</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../input/input.html" class="sidebar-link">Input</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../list/list.html" class="sidebar-link">List</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../modal/modal.html" class="sidebar-link">Modal</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../navigation/navigation.html" class="sidebar-link"
+              >Navigation</a
+            >
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../rating/rating.html" class="sidebar-link">Rating</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../text-utilities/text-utilities.html" class="sidebar-link"
+              >Text Utilities</a
+            >
+          </li>
+          <li
+            class="sidebar-item mp-m active-component"
+            style="background-color: var(--toast-color)"
+          >
```

Comment:
Why inline styling, you can avoid it right?

Reply:
No.. unfortunately 😅